Libplanet.Tests/Blockchain/Policies/BlockPolicyTest.cs

```diff
@@ -89,6 +112,123 @@ public void GetNextBlockDifficulty()
             );
         }
 
+        [Fact]
+        public void ValidateNextBlock()
+        {
+            _blockChain.Append(_genesis);
```

Comment:
Does not `BlockChain<T>.Append()` method in itself call `IBlockPolicy<T>.ValidateNextBlock()` method under the hood?  It's like not minimum reproducable code.  Could we use a plain array/list instead of `BlockChain<T>` for fixture here?